### PR TITLE
anas-res-grid-layout

### PR DESCRIPTION
**Pull request by anask601**

### CODE REVIEW

File: tasks-by-users/anask601/Responsive-grid-layout/style.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    --main-radius: 5px;
+}
+
+.container
+{
+    display: grid;
+    width: 100vw;
+    height: 100vh;
+
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/Responsive-grid-layout/style.css:13]:
```suggestion
    grid-template-columns: repeat(4, 1fr);
```

File: tasks-by-users/anask601/Responsive-grid-layout/style.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+}
+footer
+{
+    background: #ffaf87;
+    grid-area: footer;
+    
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+
```
Comment (AlishaPrasad) [inline tasks-by-users/anask601/Responsive-grid-layout/style.css:86]: you could add the following and remove the redundant lines in the above classes

nav, main, #sidebar, #content1, #content2, #content3, footer {
    padding-top: var(--main-padding);
    border-radius: var(--main-radius);
}

**Review by faran4engg — CHANGES_REQUESTED** [review]
correct folder name 😒

**Review by coder-mezab — APPROVED** [review]

File: tasks-by-users/anask601/Responsive-grid-layout/style.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+.container
+{
+    display: grid;
+    width: 100vw;
+    height: 100vh;
+
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 0.2fr 1.5fr 1.2fr 0.8fr;
```
Comment (AbbasShaikh0304) [inline tasks-by-users/anask601/Responsive-grid-layout/style.css]:
```css
    grid-template:0.2fr 1.5fr 1.2fr 0.8fr / repeat (4, 1fr);
```

File: tasks-by-users/anask601/Responsive-grid-layout/style.css (hunk 1 above)
Comment (anask601) [reply tasks-by-users/anask601/Responsive-grid-layout/style.css:13]: done

File: tasks-by-users/anask601/Responsive-grid-layout/style.css (hunk 2 above)
Comment (anask601) [reply tasks-by-users/anask601/Responsive-grid-layout/style.css:86]: done @AlishaPrasad ma'am

**Comment by AbbasShaikh0304** [comment]
Why You delete html and css files @anask601

File: tasks-by-users/anask601/Responsive-grid-layout/style.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+
+
+@media only screen and (max-width: 550px){
+    .container
+    {
+        grid-template-columns: 1fr;
+        grid-template-rows: 0.4fr 0.4fr 2.2fr 1.2fr 1.2fr 1.2fr 1fr;
```
[the remark this answers is not in the document]
Comment (anask601) [reply tasks-by-users/anask601/Responsive-grid-layout/style.css]: bcoz i have giving sidebar more space

**Review by AbbasShaikh0304 — CHANGES_REQUESTED** [review]

**Review by faran4engg — APPROVED** [review]

**Review by shaikh-7abish — APPROVED** [review]

**Review by AbbasShaikh0304 — APPROVED** [review]

**Pull request merged**